Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM python:2.7.15-slim-stretch
+FROM python:3.6-slim-stretch
```

Comment:
If you do that, it breaks the current setup. You're referencing `latest-py3` in the `docker-compose.yml`, so I don't think it's necessary to change the main `Dockerfile` just yet? `latest-py3` is setup to be built automatically by docker hub from the repo using `Dockerfile.python3`.

Reply:
The alternative would be to switch local env completely to python 3 without possibility of going back, in which case you wouldn't want to reference `latest-py3` anymore in `docker-compose.yml`, but I think it's a little early for that. Switching to python 3 by default makes sense, but we need to be able to quickly switch back, even locally.

Reply:
Okay, I wasn't sure if having an inconsistent Dockerfile to what we use in docker-compose.yml was going to cause problems.

Reply:
https://github.com/mozilla/addons-server/pull/10640/commits/00681c1aecdee381a03c1669122a9fbf1c67bb7e reverts the DockerFile changes